src/components/TimecodeTracker/tests/TimecodeTracker.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import TimecodeTracker from '../TimecodeTracker'
```

Comment:
thx for the note. I though about it and I decided, to leave it as it is at the moment, because I think TimecodeTracker is better to understand what this component is doing. Timecode as variable name will be named timeCode. ;-)